### PR TITLE
DOCS-782: Clean up Viam rover fragment formatting

### PR DESCRIPTION
**Pull request by JessamyT**
I propose that we shorten it thusly, since all that information is available to users on the Config tab anyway (as well as in the fragment itself).

### CODE REVIEW

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md [hunk 1]
```diff
@@ -33,18 +33,18 @@ Click **Save Config** to save the new configuration.
 
 {{<imgproc src="try-viam/rover-resources/fragments/fragment_configuration.png" resize="1200x" alt="Fragment configuration">}}
 
-The fragment adds the following components to your robot's JSON configuration:
+The fragment adds the following components to your robot's JSON configuration.
+For information about how you would configure a component yourself if you weren't using the fragment, click the links on each component below.
```
Comment (npentrel) [inline docs/try-viam/rover-resources/rover-tutorial-fragments.md:37]: Avoid conditionals
```suggestion
If you would like to configure each component yourself, click the respective links on each component below.
```

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md [hunk 2]
```diff
@@ -33,18 +33,18 @@ Click **Save Config** to save the new configuration.
...
+To see the pin numbers and other values specific to this fragment, [see the components on the configuration page](#see-the-components-on-the-configuration-page).
```
Comment (npentrel) [inline docs/try-viam/rover-resources/rover-tutorial-fragments.md:38]:
```suggestion
To see the configured pin numbers and other values specific to this fragment, [see the components on the configuration page](#see-the-components-on-the-configuration-page).
```

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md [hunk 3]
```diff
@@ -33,18 +33,18 @@ Click **Save Config** to save the new configuration.
...
-  * Within the board component attributes, digital interrupts: "re" to pin "37" and "le" to pin "35" and I2Cs: name "default_i2c_bus" and bus "1".
-* Right gpio [motor](/components/motor/).
-  * Within the motor attributes, board: "local", encoder: "Renc", ticks per rotation: "1992".
-  * Within the component pin assignment, type: In1/In2, A/In1: "16 GPIO 23", B/In2: "18 GPIO 24", PWM: "22 GPIO 25".
-  * Depends on local and Renc.
-* Left gpio [motor](/components/motor/).
-  * Within the motor attributes, board: "local", encoder: "Lenc", ticks per rotation: "1992".
-  * Within the component pin assignment, type: In1/In2, A/In1: "11 GPIO 17", B/In2: "13 GPIO 27", PWM: "15 GPIO 22".
-  * Depends on local and Lenc.
+- A [board component](/components/board/pi/) named `local` representing the Raspberry Pi
```
Comment (npentrel) [inline docs/try-viam/rover-resources/rover-tutorial-fragments.md:40]: This no longer has any relation to the first sentence in the paragraph. Move the two sentences you've added to below the list.

Comment (npentrel) [reply docs/try-viam/rover-resources/rover-tutorial-fragments.md:40]: Also this list is not fully accurate. It also adds a base_remote_control service, and an input controller

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md [hunk 4]
```diff
@@ -33,18 +33,18 @@ Click **Save Config** to save the new configuration.
...
+  - An I<sup>2</sup>C bus is configured on the board because the accelerometer connects to the board with I<sup>2</sup>C.
+- Two [motors](/components/motor/gpio/) (`right` and `left`)
+  - The configured pin numbers correspond to where the motor drivers are connected to the board.
+- Two [encoders](/components/encoder/single/), one for each motor
+- A wheeled [base](/components/base/), an abstraction that coordinates the motors for steering
```
Comment (npentrel) [inline docs/try-viam/rover-resources/rover-tutorial-fragments.md:45]: I think adding the following attributes here would be helpful:
* Wheel circumference (mm): 217
  * Width (mm): 260
  * Spin slip factor: 1
  
Yes they are in the configuration but without looking there they're values that aren't immediately obvious without the user doing work

Comment (npentrel) [reply docs/try-viam/rover-resources/rover-tutorial-fragments.md:45]: the same goes for the motor. Let's add those values on the list.

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md (hunk 1 above)
Comment (JessamyT) [reply docs/try-viam/rover-resources/rover-tutorial-fragments.md:37]: We already told them to add the fragment and save the config. I agree with moving these two sentences to below the list, but I think this change makes it more confusing?

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md [hunk 5]
```diff
@@ -53,28 +53,10 @@ Typically, this would suggest that they should be configured as enable pins, but
 
 {{% /alert %}}
```
Comment (JessamyT) [inline docs/try-viam/rover-resources/rover-tutorial-fragments.md:54]:
```suggestion
{{% /alert %}}

```suggestion
- An [accelerometer](/components/movement-sensor/adxl345/)

For information about how you would configure a component yourself if you weren't using the fragment, click the links on each component above.
To see the configured pin numbers and other values specific to this fragment, [see the components on the configuration page](#see-the-components-on-the-configuration-page).
```
```

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md (hunk 1 above)
Comment (JessamyT) [reply docs/try-viam/rover-resources/rover-tutorial-fragments.md:37]:
```suggestion
```

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md (hunk 2 above)
Comment (JessamyT) [reply docs/try-viam/rover-resources/rover-tutorial-fragments.md:38]:
```suggestion
```
Applying below (moving to below list)

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md (hunk 3 above)
Comment (JessamyT) [reply docs/try-viam/rover-resources/rover-tutorial-fragments.md:38]: done

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md (hunk 4 above)
Comment (JessamyT) [reply docs/try-viam/rover-resources/rover-tutorial-fragments.md:43]:
```suggestion
- A wheeled [base](/components/base/), an abstraction that coordinates the movement of the right and left motors.
The width between the wheel centers is 260mm, the wheel circumference is 217mm, and the spin slip factor is 1.
```
How about that?

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md [hunk 6]
```diff
@@ -53,28 +54,13 @@ Typically, this would suggest that they should be configured as enable pins, but
...
-* Lenc [encoder](/components/encoder/) with board: local, pins "i": "le" and depends on: local.
-* An [accelerometer](/components/movement-sensor/) with the following configuration:
-  * Model: "accel-adxl345"
-  * Name: "accelerometer"
-  * Type: "movement_sensor"
-  * and attributes of "i2c_bus": "default_i2c_bus", "use_alternate_i2c_address": false, and "board": "local".
-  * Depends on: local.
-* A microphone, type: audio_input, with attributes "audio_path_pattern": "3a" and "debug": false.
+For information about how you would configure a component yourself if you weren't using the fragment, click the links on each component above.
+To see the configured pin numbers and other values specific to this fragment, [see the components on the configuration page](#see-the-components-on-the-configuration-page).
```
Comment (npentrel) [inline docs/try-viam/rover-resources/rover-tutorial-fragments.md:58]: But if you were to configure it yourself you wouldn't have them there, how about
```suggestion
To see the configured pin numbers and other values specific to this fragment, [see the Viam Rover fragment in the Viam app](https://app.viam.com/fragment?id=3e8e0e1c-f515-4eac-8307-b6c9de7cfb84).
```

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md [hunk 7]
```diff
@@ -33,18 +33,19 @@ Click **Save Config** to save the new configuration.
 
 {{<imgproc src="try-viam/rover-resources/fragments/fragment_configuration.png" resize="1200x" alt="Fragment configuration">}}
 
-The fragment adds the following components to your robot's JSON configuration:
+The fragment adds the following components to your robot's JSON configuration.
```
Comment (npentrel) [inline docs/try-viam/rover-resources/rover-tutorial-fragments.md:36]:
```suggestion
The fragment adds the following components to your robot's JSON configuration:
```

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md [hunk 8]
```diff
@@ -33,18 +33,19 @@ Click **Save Config** to save the new configuration.
...
-* Right gpio [motor](/components/motor/).
-  * Within the motor attributes, board: "local", encoder: "Renc", ticks per rotation: "1992".
-  * Within the component pin assignment, type: In1/In2, A/In1: "16 GPIO 23", B/In2: "18 GPIO 24", PWM: "22 GPIO 25".
-  * Depends on local and Renc.
-* Left gpio [motor](/components/motor/).
-  * Within the motor attributes, board: "local", encoder: "Lenc", ticks per rotation: "1992".
-  * Within the component pin assignment, type: In1/In2, A/In1: "11 GPIO 17", B/In2: "13 GPIO 27", PWM: "15 GPIO 22".
-  * Depends on local and Lenc.
+- A [board component](/components/board/pi/) named `local` representing the Raspberry Pi
+  - An I<sup>2</sup>C bus is configured on the board because the accelerometer connects to the board with I<sup>2</sup>C.
```
Comment (npentrel) [inline docs/try-viam/rover-resources/rover-tutorial-fragments.md:39]:
```suggestion
  - An I<sup>2</sup>C bus for connection to the accelerometer.
```

**Review by npentrel — APPROVED** [review]
LGTM % some remaining small suggestions

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md (hunk 8 above)
Comment (JessamyT) [reply docs/try-viam/rover-resources/rover-tutorial-fragments.md:39]: Ooh yeah that is better :D

File: docs/try-viam/rover-resources/rover-tutorial-fragments.md (hunk 6 above)
Comment (JessamyT) [reply docs/try-viam/rover-resources/rover-tutorial-fragments.md:58]: Ooh I didn't mean for people to actually configure things themselves, just as a hypothetical. I guess this works too though?

**Pull request merged**